go.mod

```diff
@@ -139,7 +139,7 @@ require (
 	github.com/rancher/remotedialer v0.4.0
 	github.com/rancher/rke v1.7.1-rc.2
 	github.com/rancher/shepherd v0.0.0-20241023152838-da6b36b3b5b4
-	github.com/rancher/steve v0.0.0-20241029132712-2175e090fe4b
+	github.com/rancher/steve v0.0.0-20241206235902-65129458a74a
```

Comment:
@tomleb are we still supposed to use commit ids here, or did we switched to tagged releases?

Reply:
Only a subset of the repos have been switched to tagged releases yet but I'm hoping to have all of them like this week (will see how busy I get). Hopefully once code freeze is done we'll be able to make the switch from commits ids to tags.

Reply:
Good. Then, this is fine